facebook/react-native
# [Android] Add DisplayP3 background and border support to Android View

## Summary:

This builds on previous PRs for the wide gamut color [RFC](https://github.com/react-native-community/discussions-and-proposals/pull/738) and extends Android views with support for DisplayP3 backgrounds and borders.

## Changelog:

[ANDROID] [ADDED] - Add DisplayP3 background and border support to Android View

## Test Plan:

1. Pull changes from these previous PRs:
https://github.com/facebook/react-native/pull/42831
https://github.com/facebook/react-native/pull/43031
https://github.com/facebook/react-native/pull/43036
2. Follow the test steps for each PR
3. Pull these changes and build RNTester for Android: `cd packages/rn-tester && yarn android`
4. Navigate to ViewExample and observe DisplayP3 background/borders

![Screenshot_20240131-100112](https://github.com/facebook/react-native/assets/1944151/9f242212-05d1-4858-af7d-d62f21cc3cbe)
